zips/draft-ecc-lockbox-disbursement.md

```diff
@@ -63,16 +63,19 @@ additional output to a 2-of-3 P2SH multisig with keys held by the following
 "Key-Holder Organizations": Zcash Foundation, the Electric Coin Company,
 and Shielded Labs.
 
-Let $v$ be the zatoshi amount in the Deferred Dev Fund Lockbox at the
-activation height. ($v$ can be predicted in advance given that height.)
-
-The additional coinbase output MUST follow the same consensus rules as apply to
-funding stream outputs [^zip-0207-consensus-rules]. That is, the coinbase
-transaction MUST contain at least one output that pays $v$ zatoshi, in the
-prescribed way defined in ZIP 207, to the above P2SH multisig address. $v$
-zatoshi are added to the transparent transaction value pool to fund this
-output, and subtracted from the balance of the Deferred Dev Fund Lockbox (i.e.
-the latter balance is reset to zero).
+Let $v$ be the zatoshi amount in the Deferred Dev Fund Lockbox as of the end
+of the block preceding the activation height. ($v$ can be predicted in advance
+given that height.)
+
+The additional coinbase output MUST contain at least one output that pays
+$v$ zatoshi to the above P2SH multisig address, using a standard P2SH script
+of the form $\texttt{OP\_HASH160}$ $\mathsf{RedeemScriptHash}(\mathsf{height})$
```

Comment:
The reference for this was not copied over from ZIP 207.

Reply:
Also, this is just incorrect. The P2SH address is not changing with height, so this should just be a function that takes the P2SH address and returns its script hash.